tests/test_search.py

```diff
@@ -0,0 +1,1036 @@
+
+    assert client.index_name == info['index_name']
+    assert num_docs == int(info['num_docs'])
+
+    res = client.search("henry iv")
+    assert isinstance(res, Result)
+    assert (225 == res.total)
+    assert (10 == len(res.docs))
+    assert (res.duration > 0)
+
+    for doc in res.docs:
+        assert doc.id
+        assert (doc.play == 'Henry IV')
+        assert (len(doc.txt) > 0)
+
+    # test no content
+    res = client.search(Query('king').no_content())
+    assert (194 == res.total)
+    assert (10 == len(res.docs))
+    for doc in res.docs:
+        assert ('txt' not in doc.__dict__)
+        assert ('play' not in doc.__dict__)
+
+    # test verbatim vs no verbatim
+    total = client.search(Query('kings').no_content()).total
+    vtotal = client.search(Query('kings').no_content().verbatim()).total
+    assert (total > vtotal)
+
+    # test in fields
+    txt_total = client.search(Query('henry').no_content().limit_fields('txt')).total
+    play_total = client.search(Query('henry').no_content().limit_fields('play')).total
+    both_total = client.search(Query('henry').no_content().limit_fields('play', 'txt')).total
+    assert (129 == txt_total)
+    assert (494 == play_total)
+    assert (494 == both_total)
+
+    # test load_document
+    doc = client.load_document('henry vi part 3:62')
+    assert (doc is not None)
+    assert ('henry vi part 3:62' == doc.id)
+    assert (doc.play == 'Henry VI Part 3')
+    assert (len(doc.txt) > 0)
+
+    # test in-keys
+    ids = [x.id for x in client.search(Query('henry')).docs]
+    assert (10 == len(ids))
+    subset = ids[:5]
+    docs = client.search(Query('henry').limit_ids(*subset))
+    assert (len(subset) == docs.total)
+    ids = [x.id for x in docs.docs]
+    assert (set(ids) == set(subset))
+
+    # self.assertRaises(redis.ResponseError, client.search, Query('henry king').return_fields('play', 'nonexist'))
+
+    # test slop and in order
+    assert (193 == client.search(Query('henry king')).total)
+    assert (3 == client.search(Query('henry king').slop(0).in_order()).total)
+    assert (52 == client.search(Query('king henry').slop(0).in_order()).total)
+    assert (53 == client.search(Query('henry king').slop(0)).total)
+    assert (167 == client.search(Query('henry king').slop(100)).total)
+
+    # test delete document
+    client.add_document('doc-5ghs2', play='Death of a Salesman')
+    res = client.search(Query('death of a salesman'))
+    assert (1 == res.total)
+
+    assert (1 == client.delete_document('doc-5ghs2'))
+    res = client.search(Query('death of a salesman'))
+    assert (0 == res.total)
+    assert (0 == client.delete_document('doc-5ghs2'))
+
+    client.add_document('doc-5ghs2', play='Death of a Salesman')
+    res = client.search(Query('death of a salesman'))
+    assert (1 == res.total)
+    client.delete_document('doc-5ghs2')
+
+
+"""
+seems like this command was removed from redisearch
+https://github.com/RediSearch/RediSearch/pull/1372
+
```

Comment:
This is the command that seems to be removed from RediSearch

Reply:
Yank it then, just double-check.